homeworks/07-mars-viewer/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { Suspense } from 'react';
+import { Redirect, Route, Switch } from 'react-router';
+import { Loading } from './app/components/Loading/Loading';
+import { Navbar } from './app/components/Navbar/Navbar';
+import { FAVORITES_ROUTE, SOLS_ROUTE } from './app/route';
+
+const Sols = React.lazy(() => import('./features/pages/sols/Sols'));
+const Favorites = React.lazy(
+  () => import('./features/pages/favorites/Favorites')
+);
+
+function App(): React.ReactElement {
+  return (
+    <>
+      <Navbar />
+      <main>
+        <Suspense fallback={<Loading />}>
+          <Switch>
```

Comment:
note that for routing usage your server should be able to redirect all requests on your index.html for spa this link could not be viewed after refresh for example http://mars-photos-viewer.surge.sh/sols Here it could be solved by changing router on hash router